types/ramda/index.d.ts

```diff
@@ -1370,6 +1370,12 @@ export function pathOr<T>(defaultValue: T, path: Path, obj: any): T;
 export function pathOr<T>(defaultValue: T, path: Path): (obj: any) => T;
 export function pathOr<T>(defaultValue: T): _.F.Curry<(a: Path, b: any) => T>;
 
+/**
+ * Retrieves the values at given paths of an object.
+ */
+export function paths<T>(paths: Path[], obj: any): T[] | undefined;
```

Comment:
I'd love to type it better, but I just don't see any plausible alternative to having the user template the result using `T` as e.g. `number|string`, for when they expect the `paths` to resolve to both numbers and strings...

Reply:
Because of the nature of it I don't think there is any better alternative and I wouldn't complain about having to work with this.